CryptoTechReminderSystem.Main/Program.cs

```diff
@@ -1,43 +1,97 @@
 ﻿using System;
-using System.Collections.Generic;
+using System.Threading;
 using CryptoTechReminderSystem.Boundary;
 using CryptoTechReminderSystem.Gateway;
 using CryptoTechReminderSystem.UseCase;
 using dotenv.net;
+using FluentScheduler;
 
 namespace CryptoTechReminderSystem.Main
 {
-    class Program
+    class Clock : IClock
     {
-        static void Main(string[] args)
+        public DateTimeOffset Now()
+        {
+            return DateTimeOffset.Now;
+        }
+    }
+
+    public class Program : Registry
+    {
+        private static void Main(string[] args)
         {    
             DotEnv.Config(false);
-            
-            var remindDeveloper = new SendReminder(
-                new SlackGateway(
-                    "https://slack.com/",
-                    Environment.GetEnvironmentVariable("SLACK_TOKEN")
-                )
+            JobManager.Initialize(new ReminderRegistry());
+            SpinWait.SpinUntil(() => false);
```

Comment:
Why do we need this?

Reply:
It stops the main from exiting when all the schedules are setup. We want the program to keep running.

Reply:
Agreed with Robert!